python/paddle/fluid/framework.py

```diff
@@ -981,8 +981,9 @@ def __init__(self,
             if type is None:
                 raise ValueError(
                     "`type` to initialized an Operator can not be None.")
-            self.iop = core.OpBase(type)
```

Comment:
no need for clear op then

Reply:
Yes. But I keep clear ops now, because some function may still creates backward ops, like gradient clipping, optimize.minimize.